frontend/views/containers/chatroom/fakeStore.js

```diff
@@ -8,17 +8,13 @@ import {
   STATUS_EXPIRED,
   STATUS_CANCELLED
 } from '@model/contracts/voting/constants.js'
+import { MESSAGE_TYPES } from '@model/contracts/constants.js'
 
 export const chatTypes = {
   INDIVIDUAL: 'INDIVIDUAL',
   GROUP: 'GROUP'
 }
 
-export const messageTypes = {
-  INTERACTIVE: 'INTERACTIVE',
-  NOTIFICATION: 'NOTIFICATION'
-}
-
```

Comment:
Shouldn't this entire file be deleted since it's no longer needed?

Reply:
Removed this `fakeStorage.js` file. I found two parts which use some data from `fakeStore.js` so just added only those necessary data in the specific components.